launcher/spec.go

```diff
@@ -544,8 +544,6 @@ func (launcher *Launcher) createRuntimeSpec(instance *runtimeInstanceInfo) (*run
 	spec.bindHostDirs(launcher.config.WorkingDir)
 	spec.setNamespacePath(runtimespec.NetworkNamespace, launcher.networkManager.GetNetnsPath(instance.InstanceID))
 	spec.mergeEnv(createAosEnvVars(instance))
-	instance.overrideEnvVars = launcher.getInstanceEnvVars(instance.InstanceInfo)
```

Comment:
Commit message: header should contain info about what inside the commit. So, in this case it should be something like:

```go
[launcher] Fix override env var if entry exists

Detailed explanation if required.

Signed-off-by: Mykhailo Lohvynenko <Mykhailo_Lohvynenko@epam.com>
```

Reply:
applied suggestion